tools/audio/audio_tablegen.c

```diff
@@ -290,19 +304,330 @@ tablegen_soundfonts(const char *sf_hdr_out, char **soundfonts_paths, int num_sou
     return EXIT_SUCCESS;
 }
 
+/* Sequences */
+
+struct seq_order {
+    size_t num_sequences;
+    char **names;
+    char **enum_names;
+    bool *ptr_flags;
+    void *buf_ptr;
+};
+
+static void
+read_seq_order(struct seq_order *order, const char *path)
+{
+    // Read from file, we assume the file has been preprocessed with cpp so that whitespace is collapsed and each line
+    // has the form:
+    // (name,enum_name) or *(name,enum_name)
+    UNUSED size_t data_size;
```

Comment:
does taking the address of a variable not count as using it?

Reply:
The compiler doesn't warn but it felt a little weird not to since semantically it is unused in that it's never read from, but the compiler can't tell it's set-but-unused since the address is taken when calling an extern function